@vates/nbd-client/index.js

```diff
@@ -313,8 +313,8 @@ module.exports = class NbdClient {
       const exportSize = this.#exportSize
       const chunkSize = 2 * 1024 * 1024
       indexGenerator = function* () {
-        const nbBlocks = Math.ceil(exportSize / chunkSize)
-        for (let index = 0; index < nbBlocks; index++) {
+        const nbBlocks = Math.ceil(Number(exportSize / BigInt(chunkSize)))
```

Comment:
```console
> 2 / 1n
Uncaught TypeError: Cannot mix BigInt and other types, use explicit conversions
```

Reply:
how did you get this error ?

Reply:
I didn't get the error, I just tried dividing a number by a bigint.

But I misread the code, it's not what you are doing here, therefore your can ignore my comment, sorry :)